src/main/scala/sam/sceval/EvaluationPimps.scala

```diff
@@ -16,6 +16,9 @@ import scala.reflect.ClassTag
 // preaggregated to then allow for a reduceBy(model).
 // Will be useful for evaluating matching algorithms or tiny-cluster clustering problems
 
```

Comment:
@gm-spacagna  Not sure I understand the motivation for splitting this up.  Only motivation I can see is that the file is a wee bit large.  I'd rather leave as is, until it really grows to be unmanagable.